core/src/main/java/io/grpc/internal/DnsNameResolver.java

```diff
@@ -315,6 +315,12 @@ public void run() {
 
           @SuppressWarnings("unchecked")
           Map<String, ?> verifiedRawServiceConfig = (Map<String, ?>) rawServiceConfig.getConfig();
+          Map<String, ?> healthCheckedService =
+              ServiceConfigUtil.getHealthCheckedService(verifiedRawServiceConfig);
```

Comment:
This doesn't look right. I think we want this to be in ManangedChannelImpl so that it applies to all name resolvers.

What made me notice this is that RESOLUTION_RESULT_ATTR_HEALTH_CHECKING_CONFIG was put in NameResolver and not LoadBalancer. I expected it in LoadBalancer and for the NameResolver to be ignorant of it.

Reply:
moved to LoadBalancer, PTAL